pkg/vpcmodel/abstractVPC.go

```diff
@@ -89,6 +89,8 @@ type FilterTrafficResource interface {
 	// RulesInConnectivity get the list of rules of a given filter that contributes to the connection between src and dst
 	// todo: currently implemented only to sg; likely src and dst will be VPCResourceIntf instead of Node
 	RulesInConnectivity(src, dst Node, isIngress bool) ([]RulesInFilter, error)
+	// IsDefault returns true iff all sgs/nacls/.. are the default ones
+	IsDefault() bool
```

Comment:
Why do we need to expose `IsDefault()` in the level of SGLayer or NACLLayer? 
In the explainability context it is relevant to check for a certain resource(s) involved in the two input nodes connectivity only, and not for the entire layer of all such resources..

Reply:
Not sure. We might want to inform the user in the future that all relevant rules are default. Anyways, since the above functions are not actually used at the moment I removed them.